### DISCUSSION TITLE
Sentence BERT Implementation

### DISCUSSION THREAD

**Discussion by sridharkonduri-99** [jishnujayakumar/MLRC2020-EmbedKGQA#11] [Q&A]
I have been trying to improve on the ACL 2020 paper as well, I found the idea of SBERT interesting, however I couldn't find an implementation of it in the Git repo. Do you have an references or implementation steps for it?

**Comment by jishnujayakumar** [comment]
AFAIK, there are following two packages available.

- https://github.com/UKPLab/sentence-transformers
- https://huggingface.co/sentence-transformers/bert-base-nli-mean-tokens

Do mention, if you find any alternatives.

PS: Try using [SBERT-WK](https://github.com/BinWang28/SBERT-WK-Sentence-Embedding), an improved version of SBERT.

**Reply by yaowang1112** [reply]
When I use SBERT, problems appear like below.
/opt/conda/conda-bld/pytorch_1607370156314/work/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [136,0,0], thread: [64,0,0] Assertion `srcIndex < srcSelectDimSize` failed.
/opt/conda/conda-bld/pytorch_1607370156314/work/aten/src/ATen/native/cuda/Indexing.cu:658: indexSelectLargeIndex: block: [136,0,0], thread: [65,0,0] Assertion `srcIndex < srcSelectDimSize` failed.

**Reply by sridharkonduri-99** [reply]
This usually happens whens you try to access out of bound element, maybe if you can show the code snippet the problem can isolated and solved
